src/main/java/com/google/sps/data/ForumElement.java

```diff
@@ -0,0 +1,24 @@
+package com.google.sps.data;
+
+/** A comment or question element of the forum. */
+public final class ForumElement {
+
+  private final long id;
+  private final long parentId;
+  private final String topic;
+  private final long timestamp;
+  private final long likes;
+  private final String text;
+  // private final long userId
+
+  public ForumElement(
+      long id, long parentId, String topic, long timestamp, long likes, String text) {
```

Comment:
I think likes and ids can just be ints. I think ids fit within int range, and i dont think we'll be getting anywhere near 2 billion likes (max bound of int in java)

Reply:
The id is originally outputted as a long not a String.  I think you may be thinking of the UserService id which is stored as a String.  So I will probably keep the id a long but I will try to change the likes to be int.

Reply:
I store the likes as an int now.  Thanks for catching that.